charts/longhorn-1.1/longhorn-crd/1.1.200+up1.1.2/Chart.yaml

```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```

Comment:
forgot to mention that we do not need to add the annotation to the CRD charts and non-user-facing chart.

@cbron @aiyengar2 Please correct me if I am wrong.

Reply:
Yes we don't need to add it to those.